CrowdinSDK/Classes/CrowdinSDK/Utils/Attributes/AttributedTextFormatter.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  AttributedTextFormatter.swift
+//  CrowdinSDK
+//
+//  Created by Nazar Yavornytskyy on 2/19/21.
+//
+
+import Foundation
+import BaseAPI
+
+enum AttributedTextFormatter {
```

Comment:
Why it's enum?

Reply:
Thanks, TYPO - make it as struct as well